docs/content/en/docs/pipeline/chain-security.md

```diff
@@ -0,0 +1,370 @@
+    --type=kubernetes.io/dockerconfigjson \
+    -n kurator-pipeline
+```
+
+### Configuring Tekton Chains Parameters
+
+To ensure proper functioning of Tekton Chains components, apply the following configurations:
+
+```console
+kubectl patch configmap chains-config -n tekton-chains -p='{"data":{"artifacts.taskrun.format": "slsa/v1"}}'
+kubectl patch configmap chains-config -n tekton-chains -p='{"data":{"artifacts.taskrun.storage": "oci"}}'
+kubectl patch configmap chains-config -n tekton-chains -p='{"data":{"artifacts.oci.storage": "oci"}}'
+kubectl patch configmap chains-config -n tekton-chains -p='{"data":{"transparency.enabled": "true"}}'
+```
+
+With these steps completed, the preliminary setup is ready, paving the way for secure and efficient pipeline operations for supply chain security.
+
+## Triggering the Test Pipeline
+
+### Creating Kurator Pipeline Test Example
+
+Create a pipeline that includes an image task using the following command:
+
+```console
+echo 'apiVersion: pipeline.kurator.dev/v1alpha1
+kind: Pipeline
+metadata:
+  name: quick-start
+  namespace: kurator-pipeline
+spec:
+  description: "this is a quick-start pipeline, it shows how to use customTask and predefined Task in a pipeline"
+  tasks:
+    - name: git-clone
+      predefinedTask:
+        name: git-clone
+        params:
+          git-secret-name: git-credentials
+    - name: cat-readme
+      customTask:
+        image: zshusers/zsh:4.3.15
+        command:
+          - /bin/sh
+          - -c
+        args:
+          - "cat $(workspaces.source.path)/README.md"
+    - name: go-test
+      predefinedTask:
+        name: go-test
+        params:
+          packages: ./...
+    - name: go-lint
+      predefinedTask:
+        name: go-lint
+        params:
+          packages: "./..."
+          flags: "--disable=errcheck,unused,gosimple,staticcheck --verbose --timeout 10m"
+    - name: build-and-push-image
+      predefinedTask:
+        name: build-and-push-image
+        params:
+          image: "ghcr.io/<username>/<image-name>"'| kubectl apply -f -
+```
+
+Replace `<username>` with your GitHub username and `<image-name>` with your image name, like `kurator-test:0.4.1`.
+
+### Exposing Services
+
+Similar to the previous pipeline, expose the services automatically created by this pipeline:
+
+```console
+kubectl port-forward --address 0.0.0.0 service/el-quick-start-listener 30002:8080 -n kurator-pipeline
+```
+
+### Configuring Webhook
+
+Set up a webhook for this pipeline as well, following the detailed process in [Setting Up Your Pipeline](https://kurator.dev/docs/pipeline/setting/).
+
+### Triggering the Pipeline
+
+To trigger the pipeline, consider pushing some content to the repository, such as a modification to the README file. 
+The information about the received event can be observed in the terminal where the port-forwarding service is running:
+
+```console
+Forwarding from 0.0.0.0:30002 -> 8080
```

Comment:
This is for your kind env, do you mention it before?

Reply:
Yes，I need add those link